packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -216,20 +243,24 @@ export class RouterliciousOrdererRestWrapper extends RouterliciousRestWrapper {
                         documentId,
                         refreshToken,
                     );
-                    return `Basic ${ordererToken.jwt}`;
+
+                    return ordererToken;
                 }
             );
         };
 
+        const newtoken = await fetchOrdererToken();
+
         const restWrapper = new RouterliciousOrdererRestWrapper(
-            logger, rateLimiter, getAuthorizationHeader, useRestLess, baseurl);
+            logger, rateLimiter, newtoken, fetchOrdererToken, getAuthorizationHeader, useRestLess, baseurl);
+
         try {
             await restWrapper.load();
```

Comment:
Same as other comment: This whole try catch is made unnecessary by line 252. RouterliciousRestWrapper.load() can be removed entirely